### PR TITLE
Make hideExplorerPanel work in newui

### LINKED ISSUE #2503: Make hideExplorerPanel=1 work again.
@ajbar7 commented on [Thu Apr 20 2017](https://github.com/TerriaJS/nationalmap/issues/480)

Looks like this was dropped in the new UI - would be useful to have the functionality back.

### PR DESCRIPTION
**Pull request by tephenavies**
Fixes #2503.

### CODE REVIEW

**Comment by tephenavies** [comment]
I'm not sure about the `getUserProperty`, but it seems to do something like what I want, without having to check for an observable and call `track` on `'hideWorkbench'` & `'hideExplorerPanel'` `terria.userProperties` from outside of `Terria.js`.

File: lib/ReactViewModels/ViewState.js [hunk 1]
```diff
@@ -118,6 +118,16 @@ export default class ViewState {
...
         }, this);
 
+        const updateIsMapFullscreen = () => {
+            this.isMapFullScreen = (terria.userProperties.hideWorkbench === '1' || terria.userProperties.hideExplorerPanel === '1');
+        };
+        this.terria.getUserProperty('hideWorkbench');
+        this.terria.getUserProperty('hideExplorerPanel');
+
+        this._userPropertiesSubscription = knockout.getObservable(terria.userProperties, 'hideWorkbench').subscribe(updateIsMapFullscreen);
+        this._userPropertiesSubscription = knockout.getObservable(terria.userProperties, 'hideExplorerPanel').subscribe(updateIsMapFullscreen);
```
Comment (AlexGilleran) [inline lib/ReactViewModels/ViewState.js]: Should these both be `_userPropertiesSubscription` ?

Comment (tephenavies) [reply lib/ReactViewModels/ViewState.js]: No 🤕

Comment (chloeleichen) [reply lib/ReactViewModels/ViewState.js]: 💅

**Pull request merged**